aggregator/src/aggregator/http_handlers/tests/report.rs

```diff
@@ -408,56 +478,218 @@ async fn upload_handler() {
         "access-control-allow-origin" => "https://example.com/"
     );
 
-    // Reports that aren't aligned to the task time precision must be rejected
```

Comment:
There's no way to alert about time precision now, so I took this test out. Time precision misalignments are going away wholesale in https://github.com/divviup/janus/issues/4019.